ui/src/shared/components/PostDetail.tsx

```diff
@@ -65,6 +65,28 @@ const DangerText = styled.span`
 `;
 
 const PostDetail: React.FC<PostDetailProps> = function (props) {
+  const craftMessage = () => {
+    const messageDialogParameters: MessageDialogParameters = {
+      title: "Contacting",
+      job_poster: props.data.created_by,
+      position: {
+        job_title: props.data.title,
+        placeholder: "Write your message here",
+      },
+      current_user: {
+        // TODO(ivanfoong) populate actual current user
```

Comment:
looks like this is hardcoded. What exactly this `current_user` is doing?

Reply:
removed as this is part of #102